src/ui/display_bagl.c

```diff
@@ -292,7 +292,7 @@ UX_STEP_NOCB(ux_display_erc4361_step,
              pnn,
              {
                  &C_icon_certificate,
-                 "Sign ERC-4361",
+                 "Sign In",
                  "message",
```

Comment:
Can we drop this `message` line?
```suggestion
                 "Sign In",
```